### DISCUSSION TITLE
is ios 16.x testing more stable using appium 2.0 beta?

### DISCUSSION THREAD

**Discussion by blaxinkun** [appium/appium#18627] [Q&A]
I am using webdriverio and Appium for testing ios apps
I noticed that with ios 16 tests are not very stable.... so...
Appium 2.0 will improve that?
"Is it worth upgrading from Appium to Appium Beta

**Comment by jlipps** [comment]
Yes, if you are not using appium 2 then you don't have the latest xcuitest drivers and you'll be using technology that is over a year old. iOS 16 does not work well (or at all) with Appium 1.x. Appium 2 beta is the only supported version of Appium anyway, so you should already be using it.
